src/libraries/System.Memory.Data/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@ViktorHofer just to confirm is this the appropriate location for package apicompat suppresions? It was just placed there by the generator.

Reply:
Yes, that's the correct location. This change was technically non breaking but APICompat filed it as the current attributes rule checks for equality. It doesn't know that replacing the Type in the JsonConverterAttribute shouldn't be breaking.